### DISCUSSION TITLE
Installing updates for A320NX

### DISCUSSION THREAD

**Discussion by yuriks59** [flybywiresim/aircraft#4967] [General]
I installed the A320NX for free directly in the simulator, without being placed in the community. I would like to understand how to update the plane now through the Fly By Wire Installer application.

**Comment by scoopscode** [comment]
So basically you installed the Marketplace edition, which is the exact same as our stable version 0.6.1. New stable versions will come through Marketplace and the installer. However, since you downloaded the Marketplace edition I believe you would be updating your stable through MP. If you ever would like to use the development version, simply delete the `flybywire-aircraft-a320-neo` folder in the Community folder then use the [Installer](https://api.flybywiresim.com/installer) to install the up to date, development version.

**Reply by ExampleWasTaken** [reply]
@yuriks59 In order to uninstall the the marketplace version of the A32NX you have to go to your content manager and uninstall it from there. You won't find it in your community folder. After doing so you can use the installer like @scoopscode described.

**Reply by yuriks59** [reply]
Yes, thank you, I realized that if the community does not have it, then you need to remove it from the content manager and install the latest version from the developer.
